### DISCUSSION TITLE
Custom Domain showing blank page

### DISCUSSION THREAD

**Discussion by kamillaallimak** [community/community#48140] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, I've been trying to configure a custom domain (somostango.se) for my website (https://github.com/kamillaallimak/somos).

I've followed the instructions both from provided by both Github, and from Loopia (my DNS provider). According to Loopia the fault should be from Github's side, since they say everything looks good and that somostango.se is pointing to my github project.

When try to confirm that my DNS record's are configured correctly (same for [www.somostango.se](http://www.somostango.se/)):

`dig somostango.se

; <<>> DiG 9.10.6 <<>> somostango.se
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 6073
;; flags: qr rd ra ad; QUERY: 1, ANSWER: 4, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;somostango.se. IN A

;; ANSWER SECTION:
somostango.se. 2840 IN A 185.199.110.153
somostango.se. 2840 IN A 185.199.111.153
somostango.se. 2840 IN A 185.199.108.153
somostango.se. 2840 IN A 185.199.109.153

;; Query time: 16 msec
;; SERVER: 79.142.240.53#53(79.142.240.53)
;; WHEN: Mon Feb 20 18:35:46 CET 2023
;; MSG SIZE rcvd: 106
`
And according to githubs's Virtual assistant "Everything looks good on somostango.se!". Which seems reasonable since kamillaallimak.github.se redirects to somostango.se.

So both Github config's and Loopi config's seems to be correct, but something must be wrong somewhere since the page somostango.se shows nothing, anyone that can help me pinpoint the problem?

Thanks in advance!

Best regards
Kamilla (n00b programmer)

**Comment by yoannchaudet** [comment]
The site renders just fine for me now so I assume you found a solution?
